### PR TITLE
Feature/add cc type

### PR DESCRIPTION
**Pull request by andihan**
Hi @andrew-dwyer, can you please review it? 

Changes:
1) add CC type dropdown in frontend (payment method step at checkout)
2) add CC type dropdown in backend (payment method of creating new order)
3) add CC type multi-select in configuration (System > Configuration > Payment Methods > PinPayments > Credit Card Types)

Best to test it in a few different projects before merging it.

Thanks.

### CODE REVIEW

File: app/code/local/Dwyera/Pinpay/etc/system.xml [hunk 1]
```diff
@@ -24,49 +24,76 @@
...
                             <source_model>adminhtml/system_config_source_yesno</source_model>
                             <!-- field position -->
-                            <sort_order>1</sort_order>
+                            <sort_order>10</sort_order>
                             <!-- do not show this field in store scope -->
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>0</show_in_store>
                         </active>
+                        <title translate="label">
```
Comment (andrew-dwyer) [inline app/code/local/Dwyera/Pinpay/etc/system.xml]: @andihan Why was everything moved around here? That's what the sort order can be used for

Comment (andihan) [inline app/code/local/Dwyera/Pinpay/etc/system.xml]: @andrew-dwyer , just wanted to make it neat. Also we won't need to change sort_order values of others for inserting a new one.

File: app/design/adminhtml/default/default/template/pinpay/form/pinpay.phtml [hunk 2]
```diff
@@ -47,6 +47,19 @@ $_remoteAddr = Mage::helper('core/http')->getRemoteAddr(false);
         </div>
     </li>
 
+    <li style="display: none" class="online" id="c">
```
Comment (andrew-dwyer) [inline app/design/adminhtml/default/default/template/pinpay/form/pinpay.phtml]: @andihan Can you add an option to make this options in both the frontend and backend? Not all merchants will want to force their customers to enter the credit card type.

Comment (andihan) [inline app/design/adminhtml/default/default/template/pinpay/form/pinpay.phtml]: That's a good idea. I'll add it.

**Comment by jonpday** [comment]
@andihan is there any work remaining on this?

**Comment by andihan** [comment]
@jonpday, no more work for this one.

**Comment by moloughlin** [comment]
@andihan, looks good. I've merged master back in to resolve conflicts with the restructured system.xml -  please confirm and merge away.

**Pull request merged**